bin/http-server

```diff
@@ -153,16 +157,35 @@ function listen(port) {
     showDotfiles: argv.dotfiles,
     mimetypes: argv.mimetypes,
     username: argv.username || process.env.NODE_HTTP_SERVER_USERNAME,
-    password: argv.password || process.env.NODE_HTTP_SERVER_PASSWORD
+    password: argv.password || process.env.NODE_HTTP_SERVER_PASSWORD,
+    headers: {}
   };
 
+  function setHeader(str) {
+    const m = /^(.+?)\s*(:\s*(.*))$/.exec(str);
+    if (!m || m.length < 4) {
+      options.headers[str] = '';
+    } else {
+      options.headers[m[1]] = m[3];
+    }
+  }
+
   if (argv.cors) {
     options.cors = true;
     if (typeof argv.cors === 'string') {
       options.corsHeaders = argv.cors;
     }
   }
 
+  if (argv.header) {
+    if (Array.isArray(argv.header)) {
+      argv.header.forEach(h => setHeader(h));
```

Comment:
note: I remember reading somewhere that `forEach` actually has a performance impact over `for`, but I'm considering that an unnecessary optimization and not requiring any change here.